template/partials/class.tmpl.partial

```diff
@@ -1,61 +1,61 @@
-  <p style="display: inline; text-transform: capitalize"> 
-    {{oe.operatorType}} Operator
-  </p>
-  {{/oe.operatorType}}
-</div>
-
-{{#oe.device}}
+{{#oe.operator.configureDevice}}
 
 <div class = "alert alert-danger">
-  This operator is a device operator. If you are using Open Ephys hardware, use an aggregate operator that corresponds to your particular headstage(s) instead. Aggregate operators confer the following benefits:
+  This is a device configuration operator. If you are using Open Ephys hardware, use an aggregate configuration operator that corresponds to your particular headstage(s) instead. Aggregate operators confer the following benefits:
   <ul style = "margin-bottom: 0">
-    <li>The <code>address</code> properties of aggregate operators are automatically configured whereas a device operators requires manual configuration of its <code>address</code> property. This improves ease of using Open Ephys hardware.</li>
-    <li>All devices that belong to a hub (which is a collection of devices i.e. a headstage or breakout board) are automatically included, excluding the possibility of omitting configuration of a device that belongs to a hub. This improves ease of using Open Ephys hardware.</li>
-    <li>The workflow is less cluttered with configuration operators as one aggregate operator can correspond to multiple device operators. This improves workflow readability.</li>
+    <li>The <code>address</code> properties of aggregate configuration operators are automatically configured whereas a device configuration operators requires manual configuration of its <code>address</code> property. This improves ease of using Open Ephys hardware.</li>
+    <li>The workflow is less cluttered with configuration operators as one aggregate configuration operator can correspond to multiple device operators. This improves workflow legibility and expedites the workflow scripting process.</li>
   </ul> 
 </div>
 
-{{/oe.device}}
+{{/oe.operator.configureDevice}}
+
+<div>
+  <h1 id="{{id}}" data-uid="{{oe.uid}}" class="text-break" style="display: inline-block;">
+    {{oe.name}}
+    {{#sourceurl}}<a class="header-action link-secondary" title="View source" href="{{sourceurl}}"><i class="bi bi-code-slash"></i></a>{{/sourceurl}}
+  </h1>
+  {{#oe.operator.type}}
+  <p style="display: inline; text-transform: capitalize"> 
+    {{oe.operator.type}} Operator
+  </p>
+  {{/oe.operator.type}}
+</div>
 
 <div>{{{oe.description}}}</div>
 
-{{#showWorkflow}}
-<div class="workflow"><p><img src="../workflows/operators/{{name.0.value}}.bonsai" alt="{{name.0.value}} Workflow"></p></div>
-{{/showWorkflow}}
+{{#oe.operator.workflow}}
 
-{{#oe.operatorType}}
+<div class="workflow"><p><img src="../workflows/operators/{{oe.name}}.bonsai" alt="{{oe.name}} Workflow"></p></div>
 
+{{#oe.operator}} 
 <h2>Inputs & Outputs</h2>
-
 {{>partials/diagram}}
+{{/oe.operator}}
 
-{{/oe.operatorType}}
-
-{{#oe.hasSubOperators}}
+{{/oe.operator.workflow}}
 
 <h2>Properties</h2>
```

Comment:
I like "Properties" simply bc I'm trying to reflect the Bonsai UI as much as possible so that what users see in the docs is what they see when they are using Bonsai

On the topic of "Sub-Operators", we should probably change that too. It sounds like a technical term, but it's not